ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/ResponseObserverTest.kt

```diff
@@ -7,9 +7,11 @@ package io.ktor.client.tests
 import io.ktor.client.plugins.observer.*
 import io.ktor.client.request.*
 import io.ktor.client.tests.utils.*
+import io.ktor.utils.io.concurrent.*
 import kotlin.test.*
 
 class ResponseObserverTest : ClientLoader() {
+    private var observerCalls by shared(0)
```

Comment:
don't you need to set it to zero before each test?

Reply:
No. It seems that for each test block a new instance is created. Check CallValidatorTest for example.